scripts/procdockerstatsd

```diff
@@ -138,10 +139,18 @@ class ProcDockerStats(daemon_base.DaemonBase):
 
     def update_processstats_command(self):
         processdata = []
-        for process in psutil.process_iter(['pid', 'ppid', 'memory_percent', 'cpu_percent', 'create_time', 'cmdline']):
+        pid_set = set()
+        for process_obj in psutil.process_iter(['pid', 'ppid', 'memory_percent', 'cpu_percent', 'create_time', 'cmdline']):
```

Comment:
>process_obj [](http://example.com/codeflow?start=12&length=11)

For extreme host with too many processes, the containers will be huge and db refresh will impact performance. We may need to sort by (CPU, mem) usage, and pick first N items.

Reply:
updated